modules/platforms/dotnet/Apache.Ignite.Core/Impl/Cache/CacheImpl.cs

```diff
@@ -252,19 +252,6 @@ public bool IsEmpty()
                 flagPartitionRecover: _flagPartitionRecover);
         }
 
-        /** <inheritDoc /> */
-        public ICache<TK, TV> WithAllowAtomicOpsInTx()
-        {
-            var target = DoOutOpObject((int)CacheOp.WithSkipStore);
```

Comment:
Looks like it's never worked for .NET (WithSkipStore instead of WithAllowAtomicOpsInTx)

Reply:
Looks so